baremaps-cli/src/main/java/org/apache/baremaps/cli/map/Dev.java

```diff
@@ -64,11 +60,13 @@ public class Dev implements Callable<Integer> {
 
   @Override
   public Integer call() throws Exception {
-    try (var dataSource = PostgresUtils.dataSource(database)) {
+    // Configure serialization
+    var objectMapper = objectMapper();
 
-      // Configure serialization
-      var objectMapper = objectMapper();
+    var tileSet = objectMapper.readValue(tileset.toFile(), Tileset.class);
```

Comment:
Not directly linked to this issue, but I realize that the dev mode does not support javascript configuration files. Here is an [example](https://github.com/apache/incubator-baremaps/blob/1bae9022ec49b192cfb7ed3db2c133c29d9d2353/baremaps-cli/src/main/java/org/apache/baremaps/cli/map/Serve.java#L73) of how the serve command works. The ConfigReader reads or executes the configuration file, then it deserializes it.